### PR TITLE
fix: add heartbeat for message pooling

### PR DESCRIPTION
**Pull request by hejkerooo**

### CODE REVIEW

File: src/services/dsb-api.service.ts [hunk 1]
```diff
@@ -297,10 +297,20 @@ export class DsbApiService {
     }
     // use setTimeout instead of setInterval so we can control the interval
     const runner = async () => {
-      await job()
+      try {
+        console.log(`Runner heartbeat for pulling messages ${new Date().toISOString()}`);
```
Comment (kim-energyweb) [inline src/services/dsb-api.service.ts:301]: `console.log(`Runner heartbeat for pulling messages: ${new Date().toISOString()}`);`

File: src/services/dsb-api.service.ts [hunk 2]
```diff
@@ -297,10 +297,20 @@ export class DsbApiService {
...
+
+        await job()
+      } catch (e) {
+        console.error(`Runner failed ${new Date().toISOString()}`, e);
```
Comment (kim-energyweb) [inline src/services/dsb-api.service.ts:305]: console.error(`Runner job failed: ${new Date().toISOString()}`, e);

File: src/services/dsb-api.service.ts [hunk 3]
```diff
@@ -297,10 +297,20 @@ export class DsbApiService {
...
+      }
+
       setTimeout(runner, interval * 1000)
     }
     runner()
+        .catch((e) => {
+          console.error('Runner failed', e)
```
Comment (kim-energyweb) [inline src/services/dsb-api.service.ts:312]: console.error(`Runner failed: ${new Date().toISOString()}`, e);

**Review by kim-energyweb — CHANGES_REQUESTED** [review]

**Review by kim-energyweb — APPROVED** [review]
LGTM

**Review by vickenliu — APPROVED** [review]

**Pull request merged**